great_expectations/core/batch_definition.py

```diff
@@ -75,6 +82,40 @@ def get_batch(self, batch_parameters: Optional[BatchParameters] = None) -> Batch
 
         return batch_list[-1]
 
+    @overload
+    def run(
```

Comment:
I don't like `run` here, but `validate` is a `pydantic` method. Open to suggestions.